plugins/TimingPartitionController.cpp

```diff
@@ -57,20 +59,22 @@ void
 TimingPartitionController::do_configure(const nlohmann::json& data)
 {
   auto conf = data.get<timingpartitioncontroller::PartitionConfParams>();
```

Comment:
Accessing the JSON again.  Where is `timingpartitioncontroller::PartitionConfParams` defined, is this from the moo schema, shouldn't that have been deleted?